api/preprints/views.py

```diff
@@ -82,7 +87,7 @@ class PreprintList(JSONAPIBaseView, generics.ListCreateAPIView, ODMFilterMixin):
 
     + `filter[<fieldname>]=<Str>` -- fields and values to filter the search results on.
 
-    Preprints may be filtered by their `id`, `title`, `public`, `tags`, `date_created`, `date_modified`, `provider`, and `subjects`
+    Preprints may be filtered by their `id`, `title`, `public`, `tags`, `date_created`, `date_modified`, and `subjects`
```

Comment:
Logic train `->`:

The front page gets most recent preprints.

Right now they are not filtered by provider as we only have osf preprints and haven't thought about branding for other.

But it stands to reason that socarxiv would only want socarxiv preprints on their front page.